PREP_SYSTEM_FOR_DIETPI.sh

```diff
@@ -121,7 +121,47 @@ apt-get dist-upgrade -y
 
 #install packages
 echo -e "CONF_SWAPSIZE=0" > /etc/dphys-swapfile
-apt-get install -y ethtool p7zip-full hfsplus iw debconf-utils xz-utils fbset wpasupplicant resolvconf bc dbus bzip2 psmisc bash-completion cron whiptail sudo ntp ntfs-3g dosfstools parted hdparm pciutils usbutils zip htop wput wget fake-hwclock dphys-swapfile curl unzip ca-certificates console-setup console-data console-common keyboard-configuration wireless-tools wireless-regdb crda --no-install-recommends
+apt-get install -y  --no-install-recommends \
+  ethtool \
+  p7zip-full \
+  hfsplus \
+  iw \
+  debconf-utils \
+  xz-utils \
+  fbset \
+  wpasupplicant \
+  resolvconf \
+  bc \
+  dbus \
+  bzip2 \
```

Comment:
archivers such as bzip2, 7z, zip, unzip should be kept in as well to make it easy for everyone to use by default.